scalafix-core/src/main/scala/scalafix/internal/config/ScalafixConfig.scala

```diff
@@ -4,14 +4,10 @@ import scala.meta._
 
 import metaconfig._
 import metaconfig.generic.Surface
-import scalafix.Versions
 import scalafix.internal.config.ScalafixConfig._
 
 case class ScalafixConfig(
-    version: String = Versions.version,
```

Comment:
this was added in https://github.com/olafurpg/scalafix/commit/3a73ba04378f2de7bc850aa4d6be9e43169c5e14 but I don't understand why